### DISCUSSION TITLE
Como se informar sobre eventos?

### DISCUSSION THREAD

**Discussion by synga** [frontendbr/forum#1245] [Discussão]
Fala gente, beleza?

Tem um tempo que to atrás de ir em algum evento, o ultimo foi o FrontInSp, mas todo evento que acontece fico sabendo bem na hora ou dias depois. As vezes de meetups consigo ver a galera falando sobre, mas normalmente é meio de semana, tarde da noite e longe, muito difícil comparecer, então me sobram eventos mesmo pra poder interagir e conhecer umas coisas legais.

Queria saber, como vocês se informam sobre eventos que estão para acontecer? Se tem algum perfil em algum lugar, algum repo, alguma coisa pra se atualizar.

Valeu.

**Comment by victornery** [comment]
Existe um site chamado [Meetup.com](http://meetup.com) que você pode filtrar por sua localização e encontrar os próximos eventos próximos a você, como muitos outros sites semelhantes! 

Estava conversando esses dias com a @ketemartinsrufino, da Nubank, e ela falou uma coisa muito sensata: em SP, a comunidade é muito fragmentada. Poderia haver uma união melhor, porque são os mesmos rostos na maioria dos eventos.

**Comment by ketemartinsrufino** [comment]
Tem o [Sympla](https://www.sympla.com.br/) também. 
Geralmente, os eventos maiores/pagos estão no sympla. No Meetup.com tem mais meetups gratuitos.
Eu, geralmente, quando vou a um evento, começo a seguir o pessoal que conheci lá, além do próprio perfil do evento (uso mais no twitter).

**Comment by ninetails** [comment]
Nos Meetups geralmente a galera deixa um espaço pro pessoal divulgar eventos, vale ir pelo networking (apesar de concordar com o @victornery que a maioria são sempre os mesmos que vão)

**Comment by felipefialho** [comment]
Temos um repositório com eventos atualizados diariamente num trabalho sensacional do @theandersonn.

Vale a pena dar uma olhada.

http://frontendbr.com.br/eventos/

**Comment by synga** [comment]
@victornery @ketemartinsrufino E já tem alguma conversa sobre isso? Sobre como unir mais? Já vi uma galera falando no Twitter sobre isso.

**Comment by theandersonn** [comment]
@synga Tá convidado a acompanhar o repositório e site.
http://frontendbr.com.br/eventos/

A medida que vc for nos meetups e começar a seguir o feeds de algumas pessoas verá os eventos vão pipocando na sua timeline, caixa de emails e etc...
